Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
I feel that, module A should only break IF AND ONLY IF the update to module C is API breaking. Otherwise, as long as a version update for module C doesn't break the relevant API that is used, even if this version update is a major version update, module A shouldn't break.

Again, for API changes, compiler should catch them. For underlying functionality changes (same API but different behavior), unit tests in module A should catch them.

However, in all honesty, a tagged release of module A shouldn't ever break, implicit dependency version updates notwithstanding, if it had already passed tests before being tagged for release, since module A is supposed to contain the full stack of dependencies required to compile it.


Another side to this thinking is: why don't we nail down the specific version of module B, so that for a given tagged release of module A, we would NEVER pull in a different version of module B, not even a minor update?

I think this discussion highlights some downsides to the way SPM handles versioning that really shouldn't happen in a production environment, and hardening versions down to the minor-est of version numbers across the entire dependency stack seems to be the only way to prevent these pitfalls from happening.